Answers.md

```diff
@@ -1,2 +1,28 @@
 # Your responses to the short answer questions should be laid out here using Mark Down.
 ### For help with markdown syntax [Go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+### 1. Describe some of the differences between `.forEach` & `.map`.
+#### Some of the differences between `.forEach` & `.map` are as follows:
+#### `.forEach` scours an array for a particular item in the array. Once it finds the index it allows us to make use of that item but it doesn't return that item the way `.map` would.
+#### `.map` iterates over an array and breaks it up into individual parts and then returns a brand new array at the end of the sequence.
```

Comment:
`map` iterates over an array and applies every element to a callback function that does something to the data in the array, then returns a new mutated (modified) array. I guess this misunderstanding just depends on what `breaks it up into individual parts` means to you.

I could just be dumb and misreading what you're saying, but it seems like you're not quite there as far as understanding what's happening in these array methods.

Let me know on slack! I'd be happy to talk about it.